packages/cloudflare/src/cli/args.ts

```diff
@@ -5,9 +5,10 @@ import { resolve } from "node:path";
 export function getArgs(): {
   skipBuild: boolean;
   outputDir?: string;
+  disableMinification?: boolean;
```

Comment:
I think we should prefer positive over negative for variable names in general. i.e. `if (!disable<>) {}` is harder to parse (at least for me).

It's a common pattern for CLI args to have `--<options>` and `--no-<options>` too.

Finally it would make this consistent with the rest of the code (`minify` and `shouldMinify`).

Reply:
i was thinking about that as well. in next-on-pages we had `disableWorkerMinfication` which can be a bit of a mouthful... 😅